calendar.php

```diff
@@ -899,7 +899,7 @@
 		{
 			$privatecheck = '';
 		}
-		$start_date = explode("-", gmdate("j-n-Y-g:i A", $event['starttime']+$event['timezone']*3600));
+		$start_date = explode("-", gmdate("j-n-Y", $event['starttime']+$event['timezone']*3600));
```

Comment:
I don't know, I haven't tested it and checked whether all time inputs are covered yet